crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1884,6 +2046,35 @@ impl ChildObjectResolver for AuthorityStore {
         }
         Ok(Some(child_object))
     }
+
+    fn get_object_received_at_version(
```

Comment:
where is this actually called from?

Reply:
All of the calls from the object runtime to receive an object call this method. E.g., https://github.com/MystenLabs/sui/blob/f225f5819ced871e968d6c0538698ab0b69b827e/sui-execution/latest/sui-move-natives/src/object_runtime/object_store.rs#L90 uses this to do the actual loading of the object while executing the transaction.

Reply:
omg I was grepping in `/crates` so I didn't see those.

Reply:
I've been there...more times than I'd care to admit